### PR TITLE
Update dependencies to include grpc_cli

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
Update dependencies to include and install grpc_cli
grpc_cli is used as part of the tests for tekton results to ensure results are stored correctly

### CODE REVIEW

**Comment by CathalOConnorRH** [comment]
> What grpc_cli is used for ? can you update the commit message to mention that

grpc_cli is used as part of the tests for tekton results to ensure results are stored correctly. I've updated pr with the same detail.

File: test/images/devenv/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM quay.io/podman/stable:v4.2.1
 RUN set -x \
     && mkdir ~/.kube \
     && mkdir -p /tmp/dockerfile \
-    && dnf install -y git-2.37.3 openssl-3.0.2 procps-ng-3.3.17 unzip-6.0 which-2.21 xz-5.2.5 \
+    && dnf install -y git-2.37.3 openssl-3.0.2 procps-ng-3.3.17 unzip-6.0 which-2.21 xz-5.2.5 libtool-2.4.6 cmake-3.22.2 gcc-c++-12.0.1 \
```
Comment (Roming22) [inline test/images/devenv/Dockerfile:6]: Please keep the package list ordered.

File: test/images/devenv/install.sh [hunk 2]
```diff
@@ -173,6 +185,12 @@ check_install() {
     shellcheck --version
     tkn version
     yq --version
+    if [[ "$(grpc_cli help)" == *"command not found"* ]]; then 
+       echo "[ERROR] Could not find grpc_cli"
```
Comment (Roming22) [inline test/images/devenv/install.sh:189]: Please redirect the error to stdout.

Comment (CathalOConnorRH) [reply test/images/devenv/install.sh:189]: updated now

File: test/images/devenv/Dockerfile (hunk 1 above)
Comment (CathalOConnorRH) [reply test/images/devenv/Dockerfile]: list ordered.

File: test/images/devenv/install.sh (hunk 2 above)
Comment (Roming22) [reply test/images/devenv/install.sh:189]: Did you forget to commit the change? I don't see the redirection in the diff.

Comment (CathalOConnorRH) [reply test/images/devenv/install.sh:189]: it's showing for me, but I've just done a rebase so maybe it will update it for you too

**Pull request merged**